src/components/Summary/AddNewScore/index.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Typography, Row, Col, Rate, Button, Form, message, Input } from 'antd';
+import PropTypes from 'prop-types';
+import createComment from 'utils/comment/comment';
+
+class AddNewScoreForm extends React.Component {
+  render() {
+    let comment_name = '';
+    let comment_text = '';
+    let formdata = {
+      author: comment_name,
+      quality: 2,
+      completeness: 2,
+      hints: 2,
+      content: comment_text,
+    };
+    return (
+      <>
+        <div>
+          {' '}
+          You can enter new comment here
+          <Form
+            onSubmit={async () => {
```

Comment:
It would be better to provide a function to handle this `onSubmit` prop.
https://3x.ant.design/components/form/#header Check the DOC, it showcases examples with sample code.